### PR TITLE
Move some files closer to where they should live.

### PR DESCRIPTION
**Pull request by richardkiss**
FIXED: move two files and factor out `IPAddress`

### Purpose:

Remove a few import cycles.

### CODE REVIEW

**Comment by richardkiss** [comment]
This PR is best reviewed one change at a time.

This PR moves two files (the first two changes), and pulls out the `IPAddress` data type to remove a subtle cyclic dependency.

File: chia/util/network.py [hunk 1]
```diff
@@ -69,7 +36,8 @@ async def create(
         hostname: str,
         port: uint16,
         routes: Iterable[web.RouteDef] = (),
-        max_request_body_size: int = 1024**2,  # Default `client_max_size` from web.Application
+        max_request_body_size: int = 1024
+        ** 2,  # Default `client_max_size` from web.Application
```
Comment (arvidn) [inline chia/util/network.py:40]: I would have expected you need parentheses when breaking up an expression like this

**Review by arvidn — DISMISSED** [review]
looks good. There are some questionable formatting changes though (and pre-commit is complaining)

**Review by AmineKhaldi — COMMENTED** [review]
Thanks. I've sent some suggestions that when (batch) applied, would make CI green.

File: chia/util/network.py [hunk 2]
```diff
@@ -235,6 +220,8 @@ def select_port(prefer_ipv6: bool, addresses: List[Any]) -> uint16:
             selected_port = port
             break
     else:
-        selected_port = addresses[0][1]  # no matches, just use the first one in the list
+        selected_port = addresses[0][
+            1
+        ]  # no matches, just use the first one in the list
```
[the remark this answers is not in the document]
Comment (richardkiss) [reply chia/util/network.py:225]: These formatting changes happened by accident. I'm going to undo them.

**Review by AmineKhaldi — APPROVED** [review]

**Review by arvidn — APPROVED** [review]

**Pull request merged**